e2e/angular-core/src/projects.test.ts

```diff
@@ -241,7 +242,14 @@ describe('Angular Projects', () => {
     );
 
     // generate a shared lib
-    runCLI(`generate @nrwl/angular:library ${sharedLib} --no-interactive`);
+    runCLI(
+      `generate @nrwl/angular:library ${sharedLib} --buildable --no-interactive`
+    );
+    runCLI(
+      `generate @nrwl/angular:library-secondary-entry-point --library=${
+        names(sharedLib).fileName
```

Comment:
nit: we could avoid having to normalize this by naming the `sharedLib` as `uniq('shared-lib')`, we don't really need to do that in e2e tests

Reply:
updated! Thanks :)